core/src/commonMain/kotlin/com/ustadmobile/core/contentjob/ContentPluginManagerImpl.kt

```diff
@@ -0,0 +1,23 @@
+package com.ustadmobile.core.contentjob
```

Comment:
Actually - there are no alternative implementations of this. There is no reason to have a separate implementation and interface. Let's just rename ContentPluginManagerImpl to ContentPluginManager and remove the interface.

Reply:
done

Reply:
Was that change pushed? Looks like it is still in the branch:
https://github.com/UstadMobile/UstadMobile/blob/de-contentjob/core/src/commonMain/kotlin/com/ustadmobile/core/contentjob/ContentPluginManagerImpl.kt